pyproject.toml

```diff
@@ -88,11 +87,11 @@ ipython = ["ipython"]
 zmq = ["pyzmq"]
 common = ["ophyd", "databroker"]
 tools = ["doct", "lmfit", "tifffile", "historydict"]
-streamz = ["streamz"]
+integrations = ["reactivex"]
```

Comment:
I would suggest requiring this as a dev dependency too by adding `bluesky[integrations]` to the dev dependencies

Reply:
Sorted in https://github.com/bluesky/bluesky/pull/1698/commits/d9cc9d685a5184e2c34d7d63d4fafe548ee29eb5